### ISSUE TITLE
Support a canary PR to get quick feedback

### ISSUE THREAD

**Issue by sledigabel** [Skyscanner/turbolift#59]
This came up during the Turbolift post mortem, having the ability to file a PR on small % of the repositories first before expanding this to all repositories.

**Comment by rnorth** [comment]
@sledigabel it was a 'pre-mortem', wasn't it 😄? FWIW the workaround is simply to comment out most of the repos in `repos.txt`. This change would presumably involve making that more automated?

**Comment by sledigabel** [comment]
Yeah it was from the pre-mortem, added a special label for this :-)

An integrated way for this would be great, given the relative idempotency of the PRs, running first in a defined subset of repositories would add good value from a relatively easy effort of implementation.
